dasharo-security/secure-boot.robot

```diff
@@ -233,6 +233,52 @@ SBO008.001 Attempt to enroll the key in the incorrect format (firmware)
     Select File In File Explorer    DB.txt
     Read From Terminal Until    ERROR: Unsupported file type!
 
+SBO009.001 Attempt to enroll the key in the incorrect format (OS)
+    [Documentation]    This test verifies that it is impossible to load
+    ...    a certificate in the wrong file format from the OS.
+    Skip If    not ${SECURE_BOOT_SUPPORT}    SBO008.001 not supported
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    SBO008.001 not supported
+    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    SBO008.001 not supported
+    Power On
+    ${sb_menu}=    Enter Secure Boot Menu And Return Construction
+    ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
+    Erase All Secure Boot Keys    ${advanced_menu}
+    Save Changes And Reset    3    5
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    # remove existing directory that contains the keys
+    Execute Linux Command    rm -rf /usr/share/secureboot/keys
+    # make the directory structure
+    Execute Linux Command    mkdir /usr/share/secureboot/keys
+    Execute Linux Command    mkdir /usr/share/secureboot/keys/PK
+    Execute Linux Command    mkdir /usr/share/secureboot/keys/KEK
+    Execute Linux Command    mkdir /usr/share/secureboot/keys/db
+    Execute Linux Command    mkdir /usr/share/secureboot/keys/dbx
+    # generate and move incorrectly formatted db keys to the right place
+    Execute Linux Command
+    ...    openssl ecparam -genkey -name secp384r1 -out db.key && openssl req -new -x509 -key db.key -out db.pem -days 365 -subj "/CN=db"
+    Execute Linux Command    mv db.key /usr/share/secureboot/keys/db/
+    Execute Linux Command    mv db.pem /usr/share/secureboot/keys/db/
+    # generate and move incorrectly formatted dbx keys to the right place
+    Execute Linux Command
+    ...    openssl ecparam -genkey -name secp384r1 -out dbx.key && openssl req -new -x509 -key dbx.key -out dbx.pem -days 365 -subj "/CN=db"
+    Execute Linux Command    mv dbx.key /usr/share/secureboot/keys/dbx/
+    Execute Linux Command    mv dbx.pem /usr/share/secureboot/keys/dbx/
+    # generate and move incorrectly formatted PK keys to the right place
+    Execute Linux Command
+    ...    openssl ecparam -genkey -name secp384r1 -out PK.key && openssl req -new -x509 -key PK.key -out PK.pem -days 365 -subj "/CN=db"
```

Comment:
same here

Reply:
Maybe we could prepare some keyword to use for this? similar to https://github.com/Dasharo/open-source-firmware-validation/pull/161/files#diff-b06ff90d8f021dfa2e18d217f3d4c1e6861d6f671893cf292b6718b80227921aR222 but with openssl that would generate certs incorrect format?

Reply:
@TomaszAIR adressed [here](https://github.com/Dasharo/open-source-firmware-validation/pull/150/commits/5811b042862c128816abe8ab4ab2a575d7eedb5b#diff-a85ad2551c590f23fcfdd0995aeb59134f1c5600bec9d6b563798c00053e51d4R259-R262), now we use the keyword 4 times.